NCARDDatabase/settings.py

```diff
@@ -42,6 +42,10 @@
     'ncard_app',
     'crispy_forms',
     'crispy_bootstrap5',
+    'django_tables2',
+    "django_filters",
+    "bootstrap5",
+
```

Comment:
If you are depending on new Python libraries, you need to add them to pip-requirements.txt otherwise they will not be installed during setup.

Reply:
already add them into the requirements

Reply:
I can't see them on this branch. You need to make sure the changes are on the JL_frontend_CSS branch and that they have been committed and pushed.